.github/workflows/module_integration.yml

```diff
@@ -133,3 +133,71 @@ jobs:
           . .venv/bin/activate
           ./cmake_build/test/test_routing_pybind
           deactivate
+  
+  test_forcing_engine_integration:
+    # TODO: Remove when integration is ready
+    if: false
+
+    name: Test Forcings Engine Integration
+    runs-on: 'ubuntu-latest'
+    env:
+      CC: mpicc
+      CXX: mpic++
+      FC: gfortran
+      F90: gfortran
+
+    steps:
+      - name: Checkout NGen
+        uses: actions/checkout@v4
+      
+      - name: Checkout Forcings Engine
+        uses: actions/checkout@v4
+        with:
+          repository: NOAA-OWP/ngen-forcing
+          ref: master
+          path: extern/ngen-forcing
+          clean: false
+
+      - name: Set up Spack
+        uses: spack/setup-spack@v2
+        with:
+          buildcache: true
+
+      - name: Install Forcings Engine Dependencies
+        run: |
+          spack install parallel-netcdf
+          spack install netcdf-c +parallel-netcdf
+          spack install netcdf-fortran
+          spack install parallelio +pnetcdf +ncint
+          spack install esmf@8.5.0 +pnetcdf
+      
+      - name: Install ESMPy
```

Comment:
Maybe revise this to build ESMF from source rather than use spack -- thoughts @PhilMiller?

Reply:
I'd lean pretty hard against that - Donald spent a fair bit of time getting ESMF working independently, so if Spack packages reliably work, we should take advantage of that.